packages/expo-image-picker/ios/EXImagePicker/EXImagePicker.m

```diff
@@ -203,7 +203,17 @@ - (void)handleImageWithInfo:(NSDictionary * _Nonnull)info
   if ([[imageURL absoluteString] containsString:@"ext=PNG"]) {
     extension = @".png";
     data = UIImagePNGRepresentation(image);
-  } else if ([[imageURL absoluteString] containsString:@"ext=GIF"]) {
+    } else if ([[imageURL absoluteString] containsString:@"ext=BMP"]) {
+      
+      if ([[self.options objectForKey:@"allowsEditing"] boolValue]) {
+        //switch to png if editing.
+        extension = @".png";
+        data = UIImagePNGRepresentation(image);
+      } else {
+        extension = @".bmp";
+        data = nil;
```

Comment:
i think this will break if a quality level is provided, need some more guarding later in this function

Reply:
Since a non nil quality level implies compression/editing, I am bundling it with the png case.

Reply:
test the case where the programmer specifies a quality level, allowsEditing=false, and the user selects a bmp -> we want to make sure we give them back a file path with the bmp data